python_ta/reporters/core.py

```diff
@@ -44,6 +44,7 @@ def to_dict(self) -> Dict:
 NO_SNIPPET = {
     "invalid-name",
     "unknown-option-value",
+    "pep8-name-violation",
```

Comment:
Overall it seems like we should actually be able to show the snippet, at least in every case except possibly the module name. In the module name case, you can pass in `node=None` and ensure that no snippet is shown when the node is `None`.

Reply:
When I try to set `node=None` in the case of the module name, I get the following errors:

Here's the error that PythonTA reports.
![image](https://github.com/pyta-uoft/pyta/assets/63933334/c1704d53-70e2-4eff-87be-db2c335a32da)

Here's the error that Python reports.
![image](https://github.com/pyta-uoft/pyta/assets/63933334/deafa24e-eb3c-4efc-80fa-d72a58eeaa5c)